docs/finality-provider-operation.md

```diff
@@ -634,4 +634,30 @@ removal from the active set.
 > ⚠️ **Critical**: Slashing is irreversible and results in
 > permanent removal from the network.
 
-<!-- TODO: PROMETHEUS METRICS -->
+### 5.5 Prometheus
+
+The finality provider exposes Prometheus metrics for monitoring your 
+finality provider. The metrics endpoint is configurable in `fpd.conf`:
```

Comment:
Should we link the collector files here so that operators can see all the metrcis? i.e., `../metrics/eots_collectors.go` and `../metrics/fp_collectors.go`?